prover.js/package.json

```diff
@@ -36,36 +36,34 @@
   "dependencies": {
     "@coral-xyz/anchor": "0.29.0",
     "circomlibjs": "^0.1.7",
-    "ffjavascript": "^0.2.62",
-    "snarkjs": "^0.7.2"
+    "ffjavascript": "^0.2.63",
+    "snarkjs": "^0.7.4"
   },
   "devDependencies": {
     "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
     "@lightprotocol/hasher.rs": "workspace:*",
     "@lightprotocol/tsconfig": "workspace:*",
     "@rollup/plugin-typescript": "^11.1.5",
-    "@solana/web3.js": "^1.87.6",
-    "@types/chai": "^4.3.9",
+    "@solana/web3.js": "^1.91.8",
+    "@types/chai": "^4.3.14",
     "@types/chai-as-promised": "^7.1.6",
-    "@types/mocha": "^10.0.3",
-    "@types/node": "^20.10.2",
-    "@typescript-eslint/eslint-plugin": "^6.7.3",
-    "@vitest/browser": "^0.34.6",
-    "chai": "^4.3.10",
-    "chai-as-promised": "^7.1.1",
+    "@types/mocha": "^10.0.6",
+    "@types/node": "^20.12.8",
+    "@typescript-eslint/eslint-plugin": "^7.6.0",
+    "@vitest/browser": "^1.6.0",
     "circomlib": "^2.0.5",
-    "eslint": "^8.50.0",
-    "eslint-config-standard-with-typescript": "^39.1.0",
+    "eslint": "8.57.0",
+    "eslint-config-love": "^53.0.0",
```

Comment:
are we using this anywhere? @sergeytimoshin

Reply:
It should be a replacement for `eslint-config-standard-with-typescript`, but we don't use it anywhere.
Good catch, thanks!